MuMuPiKPAT/test/sanjay/selector/psiPrimePiK_MC.C

```diff
@@ -5872,7 +5905,225 @@ void psiPrimePiK_MC::Terminate()
 	}
       }
 
+    TNtupleD* dataNTuple;
+    TString dataTreeName;
+    rooKeysError = kFALSE;
+
+    if (MC) {
+      dataTreeName = "AA_recoVars";
+      dataNTuple = (TNtupleD*)AlexisOut->Get("AA_recoVars");
```

Comment:
Why do you assign dataNTuple here (and two lines below) if you do it again here:
https://github.com/lecriste/Z4430/pull/1/files#diff-2aef6870c472c96f6a5aec4107f14debR5919
?

Reply:
That's a mistake, sorry!